v2/spanner-to-sourcedb/src/test/java/com/google/cloud/teleport/v2/templates/dbutils/dml/CassandraTypeHandlerTest.java

```diff
@@ -0,0 +1,478 @@
+package com.google.cloud.teleport.v2.templates.dbutils.dml;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.nio.ByteBuffer;
+import java.sql.Timestamp;
+import java.util.*;
+
+import static com.google.cloud.teleport.v2.templates.dbutils.dml.CassandraTypeHandler.*;
+import static com.google.cloud.teleport.v2.templates.dbutils.dml.CassandraTypeHandler.handleCassandraTimestampType;
+import static org.junit.Assert.*;
+
+@RunWith(JUnit4.class)
+public class CassandraTypeHandlerTest {
```

Comment:
Add more tests covering cases for NULL values for all datatypes, min and max values, incorrect value formats, etc. Please refer to these tests for forward migration for an idea on test cases to cover: https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/spanner-common/src/test/java/com/google/cloud/teleport/v2/spanner/migrations/convertors/ChangeEventTypeConvertorTest.java

Reply:
handled the same @shreyakhajanchi